Mathlib/AlgebraicTopology/SimplicialSet.lean

```diff
@@ -84,6 +84,60 @@ scoped[Simplicial] notation3 "Δ[" n "]" => SSet.standardSimplex.obj (SimplexCat
 instance : Inhabited SSet :=
   ⟨Δ[0]⟩
 
+section Product
+
+/-- `Prod X Y` is the explicit binary product of simplicial sets `X` and `Y`. -/
+def Prod (X Y : SSet) : SSet where
+  obj n := X.obj n × Y.obj n
+  map f a := (X.map f a.1, Y.map f a.2)
+
+/-- The first projection of `Prod X Y`, onto X. -/
+@[simp]
+def Prod.fst {X Y : SSet} : (Prod X Y) ⟶ X where
+  app _ a := a.1
+
+/-- The second projection of `Prod X Y`, onto Y. -/
+@[simp]
+def Prod.snd {X Y : SSet} : (Prod X Y) ⟶ Y where
+  app _ a := a.2
+
+/-- The binary fan whose point is `Prod X Y`. -/
+@[simps!]
+def binaryProductCone (X Y : SSet.{0}) : BinaryFan X Y :=
+  BinaryFan.mk (Prod.fst) (Prod.snd)
+
+@[simp]
+theorem binaryProductCone_fst (X Y : SSet) : (binaryProductCone X Y).fst = Prod.fst :=
+  rfl
+
+@[simp]
+theorem binaryProductCone_snd (X Y : SSet) : (binaryProductCone X Y).snd = Prod.snd :=
+  rfl
+
+/-- `Prod X Y` is a limit cone. -/
+@[simps]
+def binaryProductLimit (X Y : SSet) : IsLimit (binaryProductCone X Y) where
+  lift (s : BinaryFan X Y) := {
+    app := fun n x => ((s.fst).app n x, (s.snd).app n x)
+    naturality := fun j k g => by
+      ext a
+      simp [FunctorToTypes.naturality]
+  }
+  fac _ j := Discrete.recOn j fun j => WalkingPair.casesOn j rfl rfl
```

Comment:
Minor comment, here and in a few other places: we slightly prefer "\mapsto" over "=>". (In case you haven't seen it already, this is contained in the [style guide](https://leanprover-community.github.io/contribute/style.html).

Reply:
Thank you, will fix this!